pkg/sso/client.go

```diff
@@ -35,26 +35,34 @@ type ConnectionType string
 
 // Constants that enumerate the available connection types.
 const (
-	ADFSSAML         ConnectionType = "ADFSSAML"
-	ADPOIDC          ConnectionType = "ADPOIDC"
-	Auth0SAML        ConnectionType = "Auth0SAML"
-	AzureSAML        ConnectionType = "AzureSAML"
-	CyberArkSAML     ConnectionType = "CyberArkSAML"
-	DuoSAML          ConnectionType = "DuoSAML"
-	GenericOIDC      ConnectionType = "GenericOIDC"
-	GenericSAML      ConnectionType = "GenericSAML"
-	GoogleOAuth      ConnectionType = "GoogleOAuth"
-	GoogleSAML       ConnectionType = "GoogleSAML"
-	JumpCloudSAML    ConnectionType = "JumpCloudSAML"
-	MagicLink        ConnectionType = "MagicLink"
-	MicrosoftOAuth   ConnectionType = "MicrosoftOAuth"
-	OktaSAML         ConnectionType = "OktaSAML"
-	OneLoginSAML     ConnectionType = "OneLoginSAML"
-	PingFederateSAML ConnectionType = "PingFederateSAML"
-	PingOneSAML      ConnectionType = "PingOneSAML"
-	SalesforceSAML   ConnectionType = "SalesforceSAML"
-	ShibbolethSAML   ConnectionType = "ShibbolethSAML"
-	VMwareSAML       ConnectionType = "VMwareSAML"
+	ADFSSAML          ConnectionType = "ADFSSAML"
+	ADPOIDC           ConnectionType = "ADPOIDC"
+	Auth0SAML         ConnectionType = "Auth0SAML"
+	AzureSAML         ConnectionType = "AzureSAML"
+	CasSAML           ConnectionType = "CasSAML"
```

Comment:
I think in similar fashion to @maxdeviant's review we'll need to change CasSAML to CASSAML based on the API reference https://workos.com/docs/reference/api-keys#connection-connection_type

Reply:
Documentation is wrong on that one, but ADPOIDC is wrong here. Correcting and submitting.